packaging/nmstate.spec

```diff
@@ -23,7 +23,7 @@ provider support on the southbound.
 
 %package -n python3-%{libname}
 Summary:        nmstate Python 3 API library
-Requires:       NetworkManager-libnm >= 1:1.22.10
+Requires:       NetworkManager-libnm >= 1:1.26.0
```

Comment:
That would mean 0.3.4 will not be available on Fedora 32 because Fedora 32 is containing 1.22.16 too. Do you think this is a good idea?

Reply:
I am OK to get Fedora 32 out-dated.
Increase this is enforcing nmstate-0.3 branch tested on NM 1.26 for the seek of RHEL/CentOS
CI testing.

Reply:
Okay, then I am approving it. I just wanted to let you know :-)